src/space_weather/gnss_los_ground_tec_json2ioda.py

```diff
@@ -0,0 +1,308 @@
+        PRN = 401
+    else:
+        PRN = 402
+    latitudeIPP = line['seoList'][4]['obArray'][0] 
+    longitudeIPP = line['seoList'][4]['obArray'][1]
+
+    xECEFPositionGNSS = line['seoList'][5]['obArray'][0]
+    yECEFPositionGNSS = line['seoList'][5]['obArray'][1]
+    zECEFPositionGNSS = line['seoList'][5]['obArray'][2]
+
+    tec_value = line['seoList'][0]['obValue']
+    tec_error = line['seoList'][1]['obValue']
+    elevationAngleGNSS = line['seoList'][2]['obValue']
+    sensorAzimuthAngle = line['seoList'][3]['obValue']
+
+    local_data['dateTime'] = np.append(local_data['dateTime'], dateTime)
+    local_data['latitudeIPP'] = np.append(local_data['latitudeIPP'], latitudeIPP)
+    local_data['longitudeIPP'] = np.append(local_data['longitudeIPP'], longitudeIPP)
+    local_data['elevationAngleGNSS'] = np.append(local_data['elevationAngleGNSS'], elevationAngleGNSS)
+    local_data['sensorAzimuthAngle'] = np.append(local_data['sensorAzimuthAngle'], sensorAzimuthAngle)
+    local_data['totalElectronContent'] = np.append(local_data['totalElectronContent'], tec_value)
+    local_data['totalElectronContentError'] = np.append(local_data['totalElectronContentError'], tec_error)
+    local_data['xECEFPositionGNSS'] = np.append(local_data['xECEFPositionGNSS'], xECEFPositionGNSS)
+    local_data['yECEFPositionGNSS'] = np.append(local_data['yECEFPositionGNSS'], yECEFPositionGNSS)
+    local_data['zECEFPositionGNSS'] = np.append(local_data['zECEFPositionGNSS'], zECEFPositionGNSS)
+
+    local_data['latitude'] = np.append(local_data['latitude'], line['lat'])
+    local_data['longitude'] = np.append(local_data['longitude'], line['lon'])
+    local_data['xECEFPosition'] = np.append(local_data['xECEFPosition'], line['senPos'][0])
+    local_data['yECEFPosition'] = np.append(local_data['yECEFPosition'], line['senPos'][1])
+    local_data['zECEFPosition'] = np.append(local_data['zECEFPosition'], line['senPos'][2])
+    return local_data
+
+
+def convert_string_to_dateTime(time):
+
+    # convert strings of time into python dateTime object
+    try:
+        # Combine and parse the string as a datetime object
+        dtg = datetime.strptime(time, '%Y-%m-%dT%H:%M:%S.000000')
+        # Set timezone to UTC
+        dtg = dtg.replace(tzinfo=timezone.utc)
+        # Convert to Unix time in seconds since epoch
+        dateTime = np.int64(round((dtg - epoch).total_seconds()))
+
+        return dateTime
+
+    except ValueError as e:
+        # Raise an informative error if parsing fails
+        raise ValueError(f"Invalid date or time format in input '{time}': {e}")
+
+
+def init_data_dict():
+    local_data = {}              # Before assigning the output types into the above.
+    for key in extended_varDict_keys.union(meta_keys):
+        local_data[key] = []
+    return local_data
+
+
+if __name__ == "__main__":
+
+    from argparse import ArgumentParser
+
+    parser = ArgumentParser(
+        description=('Read a satwind AMV ascii/csv file from SSEC'
```

Comment:
Superficially, this description doesn't appear to be consistent with the filename or PR title.